test/integration/test.js

```diff
@@ -281,9 +289,12 @@ describe('integration', function() {
         },
         function() {
           var ravenData = iframe.contentWindow.ravenData[0];
-          console.log(ravenData.exception.values[0].type);
-          // IE10 and Edge14 ¯\_(ツ)_/¯
-          assert.match(ravenData.exception.values[0].type, /SyntaxError|undefined/);
+          // ¯\_(ツ)_/¯
+          if (isBelowIE11() || isEdge14()) {
+            assert.equal(ravenData.exception.values[0].type, undefined);
```

Comment:
😾 Sigh, I can't believe Edge still doesn't provide this (a proper `onerror` error object argument).

cc @patrickkettner

Reply:
Edge already fixed that in v15 :), but SauceLabs has some issues running the tunnel on Edge 15, so I had to use v14 for now...

Edge v14
<img width="261" alt="screen shot 2017-09-19 at 12 19 05" src="https://user-images.githubusercontent.com/1523305/30588088-b2d05fd4-9d35-11e7-841c-9bf2f19eca84.png">

Edge v15
<img width="267" alt="screen shot 2017-09-19 at 12 19 50" src="https://user-images.githubusercontent.com/1523305/30588087-b2ce721e-9d35-11e7-897b-b458e9f6eee8.png">

Chrome
<img width="266" alt="screen shot 2017-09-19 at 12 18 13" src="https://user-images.githubusercontent.com/1523305/30588089-b2d15b82-9d35-11e7-83a4-90a9860a2236.png">

Reply:
@benvinegar sorry for taking so long to reply.  @kamilogorek is right - we _did_ fix it, it just wasn't back ported to Edge 14. FWIW, we are releasing Edge 16 in less than a month (october 17th) so 14s usage will be dropping off even faster soon.

Reply:
Thanks for the insights @patrickkettner!

Reply:
🙇